code-examples/protect-page-login/react/src/App.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+// highlight-start
+import { V0alpha2Api, Configuration, Session, Identity } from '@ory/client'
+
+function App() {
+  const [session, setSession] = useState<Session | undefined>()
+  const [logoutUrl, setLogoutUrl] = useState<string | undefined>()
+
+  // Get your Ory url from .env
+  // Or localhost for local development
+  const basePath = process.env.ORY_SDK_URL || 'http://localhost:4000'
+  const ory = new V0alpha2Api(new Configuration({
+	  basePath,
+	  baseOptions: {
+		withCredentials: true
+	  }
+	}))
```

Comment:
nit: just some minor formatting issues here

Reply:
Yes, disabled prettier (I was using different setup, messed up badly sorry!) I think should be better now?